scheduler/src/cook/kubernetes/api.clj

```diff
@@ -936,7 +940,10 @@
     (.setTty container true)
     (.setStdin container true)
 
-    (set-mem-cpu-resources resources computed-mem computed-mem cpus cpus)
+    ; add memory limit if config flag set-memory-limit? is True
+    (if (get (config/kubernetes) :set-memory-limit? true)
```

Comment:
change to `(set-mem-cpu-resources resources computed-mem (when (:set-memory-limit? (config/kubernetes)) computed-mem) cpus cpus)`

Reply:
done